incubator/jaeger/templates/agent-ds.yaml

```diff
@@ -17,6 +17,10 @@ metadata:
 spec:
   template:
     metadata:
+      annotations:
+{{- if .Values.agent.podAnnotations }}
```

Comment:
The conditional will always evaluate to true since the default value is `{}`. Either remove the default or the conditional (this applies to the other instances of this pattern as well).

Reply:
discussed offline. {} evaluates to false, but if that's the case the `if` is still redundant.

but this is still wrong, what I was trying to do was have the annotations: block inside of the `if`. I've made that change in the most recent version.